EXE/subtitles.xml

```diff
@@ -114,16 +114,19 @@
 <Static AudioFile="mod\m02\m02_done_0020_trent.wav" TextArea="Cinematic">And what am I so famous for, Mr... Umm...</Static>
 <Static AudioFile="mod\m02\m02_done_0030_reichman.wav" TextArea="Cinematic">Reichmann. Admiral Reichmann, if you may.</Static>
 <Static AudioFile="mod\m02\m02_done_0040_trent.wav" TextArea="Cinematic">It's my honor, Admiral Reichmann! So, what am I so famous for?</Static>
-<Static AudioFile="mod\m02\m02_done_0050_reichman.wav" TextArea="Cinematic">Oh, Mr. Trent, lately some of the higher circles have all been talking about you! I hope Herr Wilhelm adequately pays for your work.</Static>
+<Static AudioFile="mod\m02\m02_done_0050_reichman.wav" TextArea="Cinematic">Oh, Herr Trent, lately some of the higher circles have all been talking about you! I hope Herr Wilhelm adequately pays for your work.</Static>
 <Static AudioFile="mod\m02\m02_done_0060_trent.wav" TextArea="Cinematic">I'm not complaining.</Static>
-<Static AudioFile="mod\m02\m02_done_0070_reichman.wav" TextArea="Cinematic">Modesty – a commendable benefactor, Herr Trent, but you won't be carrying this benefactor alone. I'll make sure the payment for your last mission will be doubled. All the best, Herr Trent!</Static>
+<Static AudioFile="mod\m02\m02_done_0070_reichman.wav" TextArea="Cinematic">Modesty - a commendable virtue, Herr Trent, I see virtue often taken for granted. I'll make sure the payment for your last mission will be doubled. All the best to you, Herr Trent!</Static>
 <Static AudioFile="mod\m02\m02_done_0080_reichman.wav" TextArea="Cinematic">So you think that he's the one we need?</Static>
 <Static AudioFile="mod\m02\m02_done_0090_wilham.wav" TextArea="Cinematic">Undoubtedly, Herr Admiral. He's the one who can help us accomplish all our tasks at hand.</Static>
 <Static AudioFile="mod\m02\m02_done_0100_reichman.wav" TextArea="Cinematic">Can you guarantee that?</Static>
 <Static AudioFile="mod\m02\m02_done_0110_wilham.wav" TextArea="Cinematic">Absolutely!</Static>
-<Static AudioFile="mod\m02\m02_done_0120_reichman.wav" TextArea="Cinematic">Alright, we'll make deeds of your words.</Static>
+<Static AudioFile="mod\m02\m02_done_0120_reichman.wav" TextArea="Cinematic">Very well, We implement your plan.</Static>
```

Comment:
Plans are often executed, rather than implemented.

Reply:
Sure, That can certainly change!